source/includes/_acts-api-reference.md

```diff
@@ -23,10 +23,10 @@ https://*.actsapi.intelex.com
 
 ```CSharp 
 
-//PostMan Body for Authentication Endpoint
+//JSON Body for Authentication Endpoint
 {"client_id":"Client ID Data",
 "client_secret":"Client Secret Data",
-"audience":"https://*.intelex.com/API-release",
+"audience":"https://{tenant}.intelex.com/API-release",
```

Comment:
some of these examples have dots after tenant, some dont, they need to be consitent

Reply:
Yes it has been corrected @heydrali